pkg/process/checks/container.go

```diff
@@ -95,11 +108,11 @@ func (c *ContainerCheck) Run(cfg *config.AgentConfig, groupID int32) ([]model.Me
 		return nil, nil
 	}
 
-	groupSize := len(ctrList) / cfg.MaxPerMessage
-	if len(ctrList)%cfg.MaxPerMessage != 0 {
+	groupSize := len(ctrList) / c.maxBatchSize
+	if len(ctrList)%c.maxBatchSize != 0 {
 		groupSize++
 	}
-	chunked := chunkContainers(ctrList, c.lastRates, c.lastRun, groupSize, cfg.MaxPerMessage)
+	chunked := chunkContainers(ctrList, c.lastRates, c.lastRun, groupSize, c.maxBatchSize)
```

Comment:
I love how this is renamed to `maxBatchSize`. I think it makes a lot more sense than `MaxPerMessage`

Reply:
I was also considering using `batch` in the `Default` constants in `pkg/config` but I decided to keep them more consistent with the actual config names `max_per_message` and `max_ctr_procs_per_message`.